openprescribing/templates/measure_for_one_entity.html

```diff
@@ -10,44 +10,53 @@
 {% endblock extra_css %}
 
 {% block content %}
-  <h1>{{ entity.name }}: {{ measure.name }}</h2>
-  <h2>Presentations contributing to variation</h2>
-
-  <p>In the quarter ending {{ current_at|date:"M Y" }}, the numerator in this measure was composed of the following presentations:</p>
-
-  <table id="numerator_breakdown" class="table">
-    <thead>
-      <tr>
-        <th>Presentation</th>
-        <th>Items</th>
-        <th>Quantity</th>
-        <th>Cost</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <th>Loading...</th>
-        <td></td>
-        <td></td>
-        <td></td>
-      </tr>
-    </tbody>
-  </table>
-
-  <a class="btn btn-primary" href="{% url 'measure_numerators_by_org' %}?org={{ entity.code }}&measure={{ measure.id }}&format=csv"><span class="glyphicon glyphicon-download-alt"></span> Download this data</a>
-
-  <hr>
+  <h1>
+    {{ measure.name }}
+    <br><small>{{ entity.name }}</small>
+  </h1>
+  <p>
+    <a href="{% url measures_url_name entity.code %}" class="btn btn-default">
+      View all measures for this {{ entity_type }} &rarr;
+    </a>
+  </p>
 
   <p>This measure shows how this organisation compares with its peers across NHS England. This is indicative only, and should be approached with caution. <a href='{% url 'faq' %}#measureinterpret'>Read more about measures</a>.</p>
 
   {% include '_measures_heading.html' with entity_type=entity_type %}
   {% include '_measures_panel.html' %}
 
-  <p>
-    <a href="{% url measures_url_name entity.code %}">View all measures for {{ entity.name }}</a>
-  </p>
+  <hr>
+
+  {% if measure.numerator_can_be_queried %}
```

Comment:
@inglesp Will this conditional still be correct after your numerator breakdown work is merged?

Reply:
No, it should be `measure.numerator_is_list_of_bnf_codes`.